pkg/bindings/containers/commit.go

```diff
@@ -30,5 +38,61 @@ func Commit(ctx context.Context, nameOrID string, options *CommitOptions) (entit
 	}
 	defer response.Body.Close()
 
+	if !options.GetStream() {
+		return id, response.Process(&id)
+	}
+
+	if !response.IsSuccess() {
+		return id, response.Process(err)
+	}
+	stderr := os.Stderr
+	body := response.Body.(io.Reader)
+	dec := json.NewDecoder(body)
+	for {
+		var s struct {
```

Comment:
The structs and JSON identifiers still differ.  Let's move it into a place that both can share.

Reply:
@rhatdan that's still open.